NOTICE.txt

```diff
@@ -10,12 +10,908 @@ bring it to our attention by posting an issue.
 THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
 Do Not Translate or Localize
 
-vcpkg incorporates third party material from the projects listed below.
-The original copyright notice and the license under which Microsoft received
-such third party materials are set forth below. Microsoft reserved all other
-rights not expressly granted, whether by implication, estoppel or otherwise.
+Microsoft makes certain open source code available at https://3rdpartysource.microsoft.com,
+or you may send a check or money order for US $5.00, including the product name,
+the open source component name, platform, and version number, to:
 
-1.  Catch2
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the extent
+required to debug changes to any libraries licensed under the GNU Lesser General Public License.
```

Comment:
This block is copied out of the NOTICE.txt we were deploying inside `vcpkg-ce.tgz`